docs/configuration/alerting_rules.md

```diff
@@ -11,31 +11,36 @@ to an external service. Whenever the alert expression results in one or more
 vector elements at a given point in time, the alert counts as active for these
 elements' label sets.
 
-Alerting rules are configured in Prometheus in the same way as [recording
-rules](recording_rules.md).
-
 ### Defining alerting rules
 
-Alerting rules are defined in the following syntax:
-
-    ALERT <alert name>
-      IF <expression>
-      [ FOR <duration> ]
-      [ LABELS <label set> ]
-      [ ANNOTATIONS <label set> ]
-
-The alert name must be a valid metric name.
+Alerting rules are configured in Prometheus in the same way as [recording
+rules](recording_rules.md).
 
-The optional `FOR` clause causes Prometheus to wait for a certain duration
+A example rules file with an alert would be:
+
+```yaml
+groups:
+- name: example
+  rules:
+  - alert: HighErrorRate
+    expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+    for: 10m
+    labels:
+      severity: page
+    annotations:
+      summary: High request latency
+```
+
+The optional `for` clause causes Prometheus to wait for a certain duration
 between first encountering a new expression output vector element (like an
 instance with a high HTTP error rate) and counting an alert as firing for this
 element. Elements that are active, but not firing yet, are in pending state.
 
-The `LABELS` clause allows specifying a set of additional labels to be attached
+The `labels` clause allows specifying a set of additional labels to be attached
 to the alert. Any existing conflicting labels will be overwritten. The label
 values can be templated.
 
-The `ANNOTATIONS` clause specifies another set of labels that are not
+The `annotations` clause specifies another set of labels that are not
```

Comment:
As an aside - I think? this is the only place where identifying/non-identifying labels are mentioned? Maybe a future glossary update?

Reply:
This is the place where this is documented, I don't want to glossary to end up duplicating all the other docs.

Reply:
Fair - then I guess my point would be - it's not defined? I worked out what it meant but I would suggest perhaps a sentence or two on what it means?

Reply:
That would be more something for the alertmanager documentation.